conf/checksums

```diff
@@ -4,22 +4,15 @@ fluo:1.2.0:037f89cd2bfdaf76a1368256c52de46d6b9a85c9c1bfc776ec4447d02c813fb2
 fluo_yarn:1.0.0:c6220d35cf23127272f3b5638c44586504dc17a46f5beecdfee5027b5ff874b0
 hadoop:3.3.1:2fd0bf74852c797dc864f373ec82ffaa1e98706b309b30d1effa91ac399b477e1accc1ee74d4ccbb1db7da1c5c541b72e4a834f131a99f2814b030fbd043df66
 hadoop:3.2.2:054753301927d31a69b80be3e754fd330312f0b1047bcfa4ab978cdce18319ed912983e6022744d8f0c8765b98c87256eb1c3017979db1341d583d2cee22d029
-hadoop:3.1.4:f7eff67e948e21e23d7069dc169c98fcce757f59473e1e9a0e27d69a23df88840a9776089eeae5a8e1e496753f811868fe65638b04e3388c197931cd44d86f63
 hadoop:3.0.3:db96e2c0d0d5352d8984892dfac4e27c0e682d98a497b7e04ee97c3e2019277a
 hadoop:2.10.1:2460E02CD1F80DFED7A8981BBC934C095C0A341435118BEC781FD835EC2EBDC5543A03D92D24F2DDEEBDFE1C2C460065BA1D394ED9A73CBB2020B40A8D8B5E07
-hadoop:2.9.2:3d2023c46b1156c1b102461ad08cbc17c8cc53004eae95dab40a1f659839f28a
-hadoop:2.8.5:F9C726DF693CE2DAA4107886F603270D66E7257F77A92C9886502D6CD4A884A4
-hadoop:2.7.6:f2327ea93f4bc5a5d7150dee8e0ede196d3a77ff8526a7dd05a48a09aae25669
-hadoop:2.6.5:001ad18d4b6d0fe542b15ddadba2d092bc97df1c4d2d797381c8d12887691898
+spark:3.2.0:710854DDB0617EFCB9783725F027617FFEB6CE17A161D069F4008700E0027CB87E5BF6D772A21746AEAEC0DA06703A8B257DEF72951C1073BFE0A3EBE497AF71
 spark:3.1.2:E7B580BC67356F1B36756B18A95E452307EB3825265616E399B1766124764D6E65AB4C666AD23C93E006D88B6C83A78A7D786BE981B4F20696697860A459879D
 spark:3.0.3:22F80D2D50B964FA6F87358330CCDAA4748FA1FC546272A1EFB21868DC59A815CE9BDCEE3B520B797E3740221212BAACF1A4BA5E7166D552A0990EFB9478DE32
 spark:2.4.8:84B52335834F1DC0540D5156F4CCD642A23C3CA130EBD5DC1BC8FBF1CAF0F89B783DA34DABD200A0B6385176BAD3BA4261C2C6639CAFC8CA6B8FFB8D7BB3EA72
-spark:2.3.4:0241cbe73d3ad4fec1c655d839facc16e647ff9a3825f51c78d9fb270a753da4
-spark:2.2.2:023b2fea378b3dd0fee2d5d1de6bfaf2d8349aefe7be97a9cbcf03bbacc428d7
 zookeeper:3.7.0:64e2701c019ed46f45f4d53b2c5eaeaf7cf48c342bbafa032a0819d527fc77caebfada6bdad34e5171e6d838f40b16c95b62ce378b53dfc827af2d60cec0b17c
 zookeeper:3.6.3:3f7b1b7d9cf5647d52ad0076c922e108fa956e986b5624667c493cf6d8ff09d3ca88f623c79a799fe49c72e868cb3c9d0f77cb69608de74a183b2cbad10bc827
 zookeeper:3.5.9:0e5a64713abc6f36d961dd61a06f681868171a9d9228366e512a01324806d263e05508029c94d8e18307811867cdc39d848e736c252bf56c461273ef74c66a45
-zookeeper:3.4.14:b14f7a0fece8bd34c7fffa46039e563ac5367607c612517aa7bd37306afbd1cd
```

Comment:
This should be kept. Accumulo 1.10 still uses this version, and we may need to test with it.